plugins/field-slider/tsconfig.json

```diff
@@ -11,5 +11,5 @@
     },
     // NOTE: `test/**/*` is automatically included in `blockly-scripts start`.
     // Only src matters for production builds.
-    "exclude": ["test/**/*"]
```

Comment:
I'm a bit confused about why this is necessary. Are we not `clean`ing before `build`ing? If not maybe we should be.

Also a bit curious why the [shadow block converter](https://github.com/google/blockly-samples/blob/master/plugins/shadow-block-converter/tsconfig.json) plugin is working without this change.

Reply:
Excluding `dist` and `build` is just so VS Code doesn't complain when looking at the TS Config with those files built. This doesn't impact build pipelines since `dist` and `build` are excluded from git.

Regarding `clean` and `build`, I'll usually `clean` and `build` before committing, though for dev workflow, I usually don't worry about cleaning if my changes aren't introducing or removing build artifacts. (In the past, I've setup fresh `build` calls to remove the built folder initially so I never have to think about it, though, which might be worth considering here.)